Mathlib/Data/Polynomial/Splits.lean

```diff
@@ -313,65 +326,69 @@ theorem map_rootOfSplits {f : K[X]} (hf : f.Splits i) (hfd) :
 #align polynomial.map_root_of_splits Polynomial.map_rootOfSplits
 
 theorem natDegree_eq_card_roots {p : K[X]} {i : K →+* L} (hsplit : Splits i p) :
-    p.natDegree = (p.map i).roots.card :=
+    p.natDegree = Multiset.card (p.map i).roots :=
```

Comment:
Dot notation really doesn't work here? That's very strange.

Reply:
It really doesn't work everywhere it appears, with the error message
```
invalid field notation, function 'Multiset.card' does not have argument with type (Multiset ...) that can be used, it must be explicit or implicit with a unique name
```

Reply:
> Dot notation really doesn't work here? That's very strange.

It's because `Multiset.card` is an `AddHom`